WikiDrips/ImageGenerator.swift

```diff
@@ -13,20 +13,12 @@ class ImageGenerator: Operation{
     let initials: String
     var image: UIImage?
     
-    init(initials: String){
+    init(initials: String) {
         self.initials = initials
     }
     
     override func main() {
-        if self.isCancelled {
-            print("Cancelled")
-            return
-        }
-        
-//        do {
-            self.image = UIImage.image(withInitials: initials)
-//        } catch {
-//            print(error)
-//        }
+        if isCancelled { print("Cancelled"); return }
+        image = UIImage.image(withInitials: initials)
     }
```

Comment:
@nartiles, sorry, I'm a bit confused. After reading that documentation, I'm thinking all the KVC/KVO compliance is inherited unless you override something other than main. And it seems like we only need to override other methods (start, asynchronous, executing, and finished) if we aren't using an OperationQueue to make it asynchronous. Am I wrong in thinking OperationQueue is enough?  Do I still need to set the operation to concurrent and manually execute the operations?